src/twisted/conch/ssh/transport.py

```diff
@@ -214,7 +216,7 @@ def _getMAC(
         result.key = key
         return result
 
-    def encrypt(self, blocks):
+    def _encrypt(self, blocks):
```

Comment:
SSHCiphers is kind of a public API, so for the final merge we might need to revert this change.

But I think that this API should be private